woocommerce/wc-admin
# Add TypeScript to CustomerFeedbackModal

Adds TypeScript to the `CustomerFeedbackModal` component.

Additionally, the `@types/wordpress__components` type definitions were installed. This was added to the package to resolve lint issues, and to the root to resolve build issues. Let me know if this is not the best approach.

### Detailed Testing Instructions

Smoke test the Customer Effort Score feature, for example:

- Update WooCommerce Settings
- See the snackbar appear `How easy was it to update your settings?`
- Press `Give feedback`
- Interact with the modal and send feedback.
- See the modal close and snackbar appear `Thank you for your feedback!`.

No changelog required. This has been added for the package in the fix for #6671.
